core-web/libs/edit-content/src/lib/fields/dot-edit-content-custom-field/dot-edit-content-custom-field.component.scss

```diff
@@ -0,0 +1,25 @@
+@use "variables" as *;
+
+iframe {
+    display: block;
+    width: 100%;
+}
+
+.legacy-custom-field {
+    &--fullscreen {
```

Comment:
don't do this, is hard to search if we need to debug this

Reply:
Its necessary to alternate styles
It is less expensive to change a class than to place the styles from the viewchild reference.
We have a similar "toggle" behavior on any component? Or any suggestions to handle this case?